object-store/pyproject.toml

```
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["maturin>=1.4,<1.5", "typing_extensions"]
+requires = ["maturin>=1.4,<1.6", "typing_extensions"]
```

Maybe just set to 
```suggestion
requires = ["maturin>=1.4,<2.0", "typing_extensions"]
```
so we don't need another PR with the next release

---

Done